README.md

```diff
@@ -1,24 +1,77 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Budget App
 
-Things you may want to cover:
+> The Budget app keeps track of all your tranzactions. It will allow you to save Categories and tranzactions,
+> keep track of what you have, create category, and add a tranzactions list based on what you have and who you are.
 
-* Ruby version
+## Built With
 
-* System dependencies
+- Ruby on Rails
+- PostgresSQL
 
-* Configuration
+## Getting Started
 
-* Database creation
+To get a local copy up and running follow these simple example steps.
 
-* Database initialization
+To get a local copy up and running follow these simple example steps.
 
-* How to run the test suite
+### Prerequisites
 
-* Services (job queues, cache servers, search engines, etc.)
+- A web editor like Atom or Vscode
+- Install ruby and rails
 
-* Deployment instructions
+### Setup
 
-* ...
+- Copy this link `https://github.com/gids_dadi/Budget-monitor`.
+- Get the directory that you want to clone the repository.
+- Open the command prompt in this directory.
+- Write `git clone https://github.com/gids_dadi/Budget_monitor_app`.
+- Go to the repository folder in your command prompt `cd Budget_monitor_app`.
+
+### Install
+
+- run `bundle install` to install dependencies
+
+### Usage
+
+- run `rails s` to start the local server.
+
+### Run tests
+
+- run `rspec` to run the tests.
+
+### Deployment
+View Project:(https://budget-monitor-app.herokuapp.com/)
```

Comment:
- [ ] Kindly ensure your app deployment is properly done.

  ![ryur](https://user-images.githubusercontent.com/54848363/198559673-a93b2b44-c772-4030-bb40-dd7d1f43e972.jpg)